distribution/docs/src/main/resources/_contents/_apis/application-service-api-contents.adoc

```diff
@@ -0,0 +1,131 @@
+
+=== Application Service API
+
+The Application service has multiple interfaces which are exposed on to the OSGi runtime for other applications to use.
+For more information on these interfaces, see Application Service Interfaces.
+
+==== JMX Managed Bean
+
+Some of the Application service API is exposed via JMX.
+It can either be accessed using the JMX API or from a REST-based interface created by http://jolokia.org[Jolokia] that comes with ${branding}.
+Here are the interfaces that are exposed in the Managed Bean:
+
+getApplicationTree:: Creates an application hierarchy tree that shows relationships between applications.
+startApplication:: Starts an application with the given name.
+stopApplication:: Stops an application with the given name.
+addApplications:: Adds a list of application that are specified by their URL.
+
+==== Configuration Files
+
+Support for configuration files was added to allow for an initial installation of applications on first run.
+
+==== Initial Application Installation
+
+[WARNING]
+====
+This application list configuration file is only read on first start.
```

Comment:
"...on first start." is clear to me but I worry some admin might not quite get it. Maybe be more explicit and say something like, "...the first time the server is started after being unzipped."